heroku/cli
# migrate(W-14179949): pg-v5: Upgrade pg:settings:track-functions

This PR moves `pg:settings:track-functions` to the CLI and migrates to TypeScript

### Test Instructions
1. Checkout this branch and run `yarn && yarn build && cd packages/cli
2. run `./run/bin pg:settings:track-functions -a YOUR_APP YOUR_POSTGRES all`
3. Observe the output is the same as the original.
